src/migrations/20240102142100-incoming-webhooks-created-by.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        ALTER TABLE incoming_webhooks ALTER COLUMN created_by_user_id DROP NOT NULL;
+        ALTER TABLE incoming_webhook_tokens ALTER COLUMN created_by_user_id DROP NOT NULL;
+        `,
+        cb,
+    );
+};
+
+exports.down = function (db, callback) {
+    callback();
+};
```

Comment:
What happens if you move forwards, then backwards, and then forwards again? Will this migration fail because there's no NOT NULL to drop? or will it just silently continue with life?

Reply:
Because we don't apply down migrations, the column will remain nullable. I don't see a point of manually making it non-nullable as it's more restrictive than being nullable.